src/series.jl

```diff
@@ -116,6 +116,8 @@ struct RasterSeries{T,N,D,R,A<:AbstractArray{T,N}} <: AbstractRasterSeries{T,N,D
     dims::D
     refdims::R
 end
+RasterSeries(data::DD.AbstractBasicDimArray; kw...) = 
+    RasterSeries(data, dims(data); kw...)
```

Comment:
```suggestion
RasterSeries(data::DD.AbstractBasicDimArray{<:Union{AbstractRasterStack,AbstractRaster}}; kw...) = RasterSeries(data, dims(data); 
```

Has to hold a Raster or RasterStack

Reply:
I didn't want to limit it to that, in principle it could hold strings (of filenames) as well. If it holds something that isn't allowed then `RasterSeries(data, dims(data)` will fail anyway

Reply:
Ahh right because strings will work too. Ok makes sense